awx/main/models/jobs.py

```diff
@@ -690,6 +690,9 @@ def task_impact(self):
             count_hosts = 2
         else:
             count_hosts = Host.objects.filter(inventory__jobs__pk=self.pk).count()
+            if self.job_slice_count > 1:
+                # Integer division intentional
```

Comment:
This isn't necessary, it's obvious that we want to represent a whole host rather than some fraction of a host.

Reply:
Maybe I need to document what problem this is addressing?

In `test_shard_task_impact`, we have a case of 50 forks, 3 hosts in the inventory, and 3 job slices. We know that one slice job runs on exactly 1 host. Without this modification, the task manager considers its task_impact as if it were running on 3 hosts, for a task_impact of 4. With this modification, `count_hosts` is integer 1, and task_impact is 2, as a user would expect.